lua/lsp-notify/init.lua

```diff
@@ -1,111 +1,407 @@
----@type LspNotifyConfig
-local options = nil
+--- Options for the plugin.
+---@class LspNotifyConfig
+local options = {
+  --- Function to be used for notifies.
+  --- Best works if `vim.notify` is already overwritten by `require('notify').
+  --- If no, you can manually pass `= require('notify')` here.
+  notify = vim.notify,
+
+  --- Icons.
+  --- Can be set to `= false` to disable.
+  ---@type {spinner: string[] | false, done: string | false} | false
+  icons = {
+    --- Spinner animation frames.
+    --- Can be set to `= false` to disable only spinner.
+    spinner = { "⣾", "⣽", "⣻", "⢿", "⡿", "⣟", "⣯", "⣷" },
+    --- Icon to show when done.
+    --- Can be set to `= false` to disable only spinner.
+    done = "󰗡"
```

Comment:
Can we use "✓" here, or something which will be visible without special fonts?

Reply:
Good idea